packages/templates/packages/workers.new/src/index.ts

```diff
@@ -5,11 +5,11 @@
 type Redirects = Record<string, [string, string, string, string?]>;
 
 // stackblitz repository source
-const source = "github/cloudflare/templates/tree/main";
+const source = "github/wrangler2/tree/main/packages/templates";
 
 // deploy with cloudflare source
 
-const src = "https://github.com/cloudflare/templates/tree/main";
```

Comment:
Need to do a `wrangler publish` on this Worker at some point after the PR goes in.